ansible_runner/runner.py

```diff
@@ -44,11 +44,11 @@ def event_callback(self, event_data):
         self.last_stdout_update = time.time()
         if 'uuid' in event_data:
             filename = '{}-partial.json'.format(event_data['uuid'])
-            partial_filename = os.path.join(self.config.artifact_dir,
-                                            'job_events',
-                                            filename)
-            full_filename = os.path.join(self.config.artifact_dir,
-                                         'job_events',
+            events_directory = os.path.join(self.config.artifact_dir, 'job_events')
+            if not os.path.exists(events_directory):
+                os.mkdir(events_directory, 0o700)
```

Comment:
If `ansible-playbook` doesn't actually _run_, then stdout verbose events don't recorded because this directory doesn't get created (the callback plugin has code that does this directory creation later).